tests/test_blueprint.py

```diff
@@ -430,6 +430,60 @@ def view_func():
         assert response.status_code == 200
         assert response.json == {'Value': 'OK'}
 
+    def test_blueprint_custom_docstring_separator(self, app):
+        api = Api(app)
+        blp = Blueprint('test', __name__, url_prefix='/test',
+                        docstring_sep="~~~")
+        client = app.test_client()
+
+        @blp.route('/', methods=('PUT', 'PATCH', ))
+        def view_func():
+            """Summary
+
+            Long description
+            ~~~
+            Ignore
+            """
+            return jsonify({'Value': 'OK'})
+
+        api.register_blueprint(blp)
+        spec = api.spec.to_dict()
+        path = spec['paths']['/test/']
+        for method in ('put', 'patch', ):
+            assert path[method]['summary'] == 'Summary'
+            assert path[method]['description'] == 'Long description'
+
+        response = client.put('/test/')
+        assert response.status_code == 200
+        assert response.json == {'Value': 'OK'}
```

Comment:
I probably should remove all the extra asserts from these tests and only check the strings, yeah? I guess I kept them in there to make sure that adding the arg didn't mess anything else up.

Lots of code duplication though - I'm not happy about that.